src/content/guides/partial-payment-extension.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Partial Payment
+description: An extension to the Payment Request Protocol allowing merchant integrators to accept a partial amount of a transaction through Centrapay.
+nav:
+  path: Reference/Merchant Integrations
+  order: 8
+---
+
+Partial Payment is an extension to the Payment Request Protocol that allows
+customers to pay a partial amount of a payment request using Centrapay.
+However, the customer is responsible for ensuring payment of the remaining amount.
```

Comment:
What are we trying to say here? I don't think that the meaning is coming accros very well